Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:16.04
+
+RUN mkdir /almond
+COPY . /almond
```

Comment:
Normally, I try to follow the [FHS](https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.pdf) and use a subdirectory of /opt. Also, this command should probably be after all the other apt-get stuff, to improve build caching.

Reply:
Great, will do